MobileCenter/MobileCenter/MSMobileCenter.m

```diff
@@ -249,6 +256,19 @@ - (void)setLogUrl:(NSString *)logUrl {
   }
 }
 
+- (void)setCustomProperties:(MSCustomProperties *)customProperties {
+  if (!customProperties) {
+    MSLogError([MSMobileCenter logTag], @"Custom properties may not be null");
+    return;
+  }
+  NSDictionary<NSString *, NSObject *> *properties = customProperties.properties;
+  if (properties.count == 0) {
```

Comment:
Combine with above if statement. You can just say *Custom properties may not be null or empty*